dynamiq/nodes/rankers/recency.py

```diff
@@ -73,6 +73,7 @@ class TimeWeightedDocumentRanker(Node):
     max_days: int = 3600
     min_coefficient: float = 0.9
     date_field: str = "date"
+    default_date: str = "01 January, 2022"
```

Comment:
why do we hardcore old date?

Reply:
Changed logic to update score only in case date is provided in metadata.